src/current/v23.2/physical-cluster-replication-technical-overview.md

```diff
@@ -0,0 +1,62 @@
+- Primary producer job: Protects data on the primary cluster and sends updates to the standby cluster.
+
+The stream initialization proceeds as follows:
+
+1. The standby's consumer job connects via its system interface to the primary cluster and starts the primary cluster's physical stream producer job.
+1. The primary cluster chooses a timestamp at which to start the physical replication stream. Data on the primary is protected from [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) until it is replicated to the standby using a [protected timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#protected-timestamps).
+1. The primary cluster returns the timestamp and a [job ID]({% link {{ page.version.version }}/show-jobs.md %}#response) for the replication job.
+1. The standby cluster retrieves a list of all nodes in the primary cluster. It uses this list to distribute work across all nodes in the standby cluster.
+1. The initial scan runs on the primary and backfills all data from the primary virtual cluster as of the starting timestamp of the replication stream.
+1. Once the initial scan is complete, the primary then begins streaming all changes from the point of the starting timestamp.
+
+<img src="{{ 'images/v23.2/physical-rep-to.png' | relative_url }}" alt="Two virtualized clusters with system and application tenants showing the directional stream." style="border:0px solid #eee;max-width:100%" />
+
+### During the replication stream
+
+The replication happens at the byte level, which means that the job is unaware of databases, tables, row boundaries, and so on. However, when a [cutover](#cutover-and-promotion-process) to the standby cluster is initiated, the replication job ensures that the cluster is in a transactionally consistent state as of a certain point in time. Beyond the application data, the job will also replicate users, privileges, basic zone configuration, and schema changes.
+
+{% comment %} Add:  no need to coordinate with any schema changes. Don't know anything about zone configurations will be stored using the default zone config.-->{% endcomment %}
+
+During the job, [rangefeeds]({% link {{ page.version.version }}/create-and-configure-changefeeds.md %}#enable-rangefeeds) are periodically emitting resolved timestamps (the time where the ingested data is known to be consistent). That is, a guarantee that there are no new writes from before that timestamp. This allows the standby cluster to move the [protected timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#protected-timestamps) forward as the replicated timestamp advances. This information is sent to the primary cluster, which allows for [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) to continue as the replication stream on the standby cluster advances.
```

Comment:
The first two sentences here are really complex when taken together. I wonder if it can be simplified to avoid the parenthetical and then the "That is.." further explanation.

Reply:
I have tried to simplify the sentences a little so there aren't too many intros complicating things. Happy to revise again though